src/main/java/com/redhat/devtools/intellij/lsp4ij/console/explorer/LanguageServerExplorer.java

```diff
@@ -124,10 +125,15 @@ public void invokePopup(Component comp, int x, int y) {
                         switch (processTreeNode.getServerStatus()) {
                             case starting:
                             case started:
-                                // Stop language server action
+                                // Stop and disable the language server action
                                 group = new DefaultActionGroup();
                                 AnAction stopServerAction = ActionManager.getInstance().getAction(StopServerAction.ACTION_ID);
                                 group.add(stopServerAction);
+                                if ("true".equals(System.getProperty("idea.is.internal"))) {

```

Comment:
`if (Boolean.getBoolean("idea.is.internal")) {`

Reply:
fixed